src/page-modules/contact/components/form-selector/contact-form-selector.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link';
+import { TranslatedString } from '@atb/translations';
+import { Input } from '../input';
+import style from './contact-form-selector.module.css';
+import { usePathname } from 'next/navigation';
+
+export type ContactForm = {
+  description: TranslatedString;
+  href: string;
+  page: React.ReactNode;
+};
+
+type ContactFormSelectorProps = {
+  contactForms: ContactForm[];
+};
+
+const ContactFormSelector = ({ contactForms }: ContactFormSelectorProps) => {
+  const pathname = usePathname();
+  return (
+    <nav>
+      {contactForms.map((contactForm, index) => {
+        const isActive = pathname.includes(contactForm.href);
+
+        return (
+          <Link
+            key={index}
+            shallow={true}
+            href={contactForm.href}
+            className={style.link}
+          >
+            <Input
+              label={contactForm.description}
+              type="radio"
+              checked={isActive}
+            />
+          </Link>
```

Comment:
We should not use an <input> inside of a link-element 🤔

Reply:
Okey, is there another way we can make the links appear as a set of radio buttons?🤔

Reply:
_Note_: Look into using input with shallow navigation on change (i.e. what we dicussed on Slack)